### PR TITLE
tickets/DM-30317: Added the Jenkinsfile and necessary infrastructure to build the Integ…

### PR DESCRIPTION
**Pull request by rbovill**
…rationTests documentation.

Just mostly boilerplate content for the moment, but the infrastructure is in place.

### CODE REVIEW

File: doc/user-guide/user-guide.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.. add name and email between the *'s below e.g. *Marie Smith <msmith@lsst.org>*
+
+.. _User_Guide:
+
+###########################
+IntegrationTests User Guide
+###########################
+
+.. Update links and labels below
+.. image:: https://img.shields.io/badge/GitHub-ts_athexapod-green.svg
```
Comment (mareuter) [inline doc/user-guide/user-guide.rst:46]: These all point to ATHexpod. Should be changed to point to this package.

File: doc/user-guide/user-guide.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_athexapod
+.. image:: https://img.shields.io/badge/ts_xml-ATHexapod-green.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/ATHexapod.html
+
+
+[This area should give an introduction to users, but at greater depth than the overview on the higher-level page. Some repetition is fine.
+Discussion of high-level control classes (if applicable), primary use cases, links to any useful documentation etc can be included.
+Reference to configurations may also be worthwhile]
+
+IntegrationTests Interface
```
Comment (mareuter) [inline doc/user-guide/user-guide.rst:60]: This section can be removed as this isn't a CSC.

File: doc/user-guide/user-guide.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
+..
```
Comment (mareuter) [inline doc/user-guide/user-guide.rst:1]: All of the boilerplate comments should be removed as they are just distracting.

File: doc/index.rst [hunk 4]
```diff
@@ -1,51 +1,132 @@
-.. py:currentmodule:: lsst.ts.IntegrationTests
+..
```
Comment (mareuter) [inline doc/index.rst:1]: More the boiler plate comments here since they are distracting.

File: doc/version-history.rst [hunk 5]
```diff
@@ -0,0 +1,17 @@
+.. _Version_History:
+
+===============
+Version History
+===============
+
+.. At the time of writing the Version history/release notes are not yet standardized amongst CSCs.
```
Comment (mareuter) [inline doc/version-history.rst:7]: Remove the comment and replace the version history here with the first descriptions of things you've done for the first version.

**Review by mareuter — APPROVED** [review]
Some things to clean up here, but I'm not going to ask for a rereview since we are just starting to fill out the docs.

File: doc/user-guide/user-guide.rst (hunk 1 above)
Comment (rbovill) [reply doc/user-guide/user-guide.rst]: fixed

File: doc/user-guide/user-guide.rst (hunk 2 above)
Comment (rbovill) [reply doc/user-guide/user-guide.rst]: fixed

File: doc/user-guide/user-guide.rst (hunk 3 above)
Comment (rbovill) [reply doc/user-guide/user-guide.rst]: fixed

File: doc/index.rst (hunk 4 above)
Comment (rbovill) [reply doc/index.rst]: fixed

File: doc/version-history.rst (hunk 5 above)
Comment (rbovill) [reply doc/version-history.rst:7]: fixed

**Pull request merged**